lib/shopify-cli/core/monorail.rb

```diff
@@ -104,13 +74,39 @@ def send_event(event)
               use_ssl: ENDPOINT_URI.scheme == 'https'
             ) do |http|
               post = Net::HTTP::Post.new(ENDPOINT_URI.request_uri, headers)
-              post.body = JSON.dump(event)
+              post.body = JSON.dump(build_payload(start_time, end_time, args, err))
               http.request(post)
             end
           rescue
             # silently fail on errors, fire-and-forget approach
           end
         end
+
+        def build_payload(start_time, end_time, args, err = nil)
+          {
+            schema_id: INVOCATIONS_SCHEMA,
+            payload: {
+              project_type: Project.current_project_type.to_s,
+              args: args.join(' '),
+              time_start: start_time,
+              time_end: end_time,
+              total_time:  end_time - start_time,
+              success: !err,
```

Comment:
nitpick: if you're checking for nils explicitly, use `!err.nil?`in place of general falseyness